packages/core/src/lib/code-snippet.svelte

```diff
@@ -157,8 +102,9 @@ onMount(async () => {
 
   <div class="flex gap-x-4 bg-light p-4">
     <!-- The formatting here is intentional to preserve the formatting of `code` -->
-    <pre class="flex-1 overflow-x-auto language-{language}"><code
-        bind:this={element}
+    <pre
+      class="flex-1 overflow-x-auto"
+      use:highlightCode><code
         class="language-{language}"
         data-dependencies={dependencies.join(',')}>{code}</code
```

Comment:
`data-dependencies` is both prism-specific, and handled automatically by `prism`'s autoloader. I think we can safely remove it

Reply:
good point. removed. will update usages in app to remove this prop as well